tethysext/atcore/services/resource_condor_workflow.py

```diff
@@ -18,12 +18,13 @@ class ResourceCondorWorkflow(object):
     """
     Helper class that prepares and submits the new project upload jobs and workflow.
     """
-    def __init__(self, user, workflow_name, workspace_path, resource_db_url, resource,
+    def __init__(self, app, user, workflow_name, workspace_path, resource_db_url, resource,
```

Comment:
Why are we adding the app as a parameter?

Reply:
It's needed by the downstream Workflow classes in STEEM and Shadow. You'll see in the steem_adapter PR.